### DISCUSSION TITLE
GeForce 840m, Windows 10, Vulkan, black screen/missing graphics right before the main menu.

### DISCUSSION THREAD

**Discussion by Hasster1** [ppy/osu#22958] [Q&A] [answered]
GeForce 840m, Windows 10 1909.
Tried switching to vulkan, got hit with this - 

https://user-images.githubusercontent.com/20757715/227790292-6d9badfa-2593-4706-8f70-7283766c18fe.mp4

If i alt-tab in and out, it just gets stuck on the latest frame that it showed before alt-tabbing.
I can press buttons and do stuff, so it doesn't hang or anything.

Logs:
[runtime.log](https://github.com/ppy/osu/files/11072272/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11072273/updater.log)
[network.log](https://github.com/ppy/osu/files/11072274/network.log)
[performance.log](https://github.com/ppy/osu/files/11072275/performance.log)

**Comment by bdach** [comment]
Please check if you have shadowplay enabled, and disable it if you do.

**Reply by Hasster1** [reply]
No shadowplay + It's not even supported on this GPU.

**Comment by Loreos7** [comment]
Duplicate of https://github.com/ppy/osu/discussions/22887

**Reply by bdach** [reply]
It's kinda not. Glitches != full game freeze.

**Reply by Loreos7** [reply]
Just look at videos in - they're the same. Game works after "glitch" or "frame freeze" as you can hear.

**Reply by bdach** [reply]
i have watched the videos

**Comment by bdach** [comment, marked as answer]
Vulkan is going to be pulled as a choice on all platforms for now until it is stabilised (#22961).
